lib/reline/line_editor.rb

```diff
@@ -578,8 +578,9 @@ def context
       @context
     end
 
-    def retrieve_completion_block(set_completion_quote_character = false)
-      @line_editor.retrieve_completion_block(set_completion_quote_character)
+    def retrieve_completion_block(_unused = false)
```

Comment:
I left the parameter because this is exposed to DialogProcScope like this.
```ruby
# This is possible
Reline.add_dialog_proc(:dialogname, ->{
  p retrieve_completion_block(true)
  nil
})
```

But code searching in github, it looks like this method is not used yet. I think we can remove the parameter.

I didn't confirm yet, but this method itself might be useful to use from IRB to reduce this hack
https://github.com/ruby/irb/blob/84366b8cdd7ac4c200a03405a07ed074aff5ae23/lib/irb/input-method.rb#L302-L305
Hack that stores `@completion_params` in completion_proc and read it from dialog_proc